src/mon/PGMap.cc

```diff
@@ -3513,8 +3513,13 @@ int process_pg_map_command(
       copy(dumpcontents.begin(), dumpcontents.end(),
            inserter(what, what.end()));
     }
-    if (what.empty())
-      what.insert("all");
+    if (what.empty()) {
+      if (CEPH_FEATURE_SERVER_REEF) {
```

Comment:
The reef release is quite soon while the patch changes the output of a pretty frequently used command. This makes me worried about people who scripted it.

Though the patch looks technically fine, I would recommend bring it to next CDM (as it's an interface change) and merging after Reef is out.

Reply:
Thanks @rzarzynski. I can bring this up at the next CDM!

Reply:
@rzarzynski I realize that today's CDM is at 21:00 EDT which is 2:00 AM my local time! Is there anything else I could do instead if I can't attend it?